zetaclient/orchestrator/orchestrator.go

```diff
@@ -553,7 +570,67 @@ func (oc *Orchestrator) ScheduleCctxBTC(
 		// try confirming the outbound or scheduling a keysign
 		if nonce%interval == zetaHeight%interval && !oc.outboundProc.IsOutboundActive(outboundID) {
 			oc.outboundProc.StartTryProcess(outboundID)
-			oc.logger.Debug().Msgf("ScheduleCctxBTC: sign outbound %s with value %d\n", outboundID, params.Amount)
+			oc.logger.Debug().Msgf("ScheduleCctxBTC: sign outbound %s with value %d", outboundID, params.Amount)
+			go signer.TryProcessOutbound(
+				ctx,
+				cctx,
+				oc.outboundProc,
+				outboundID,
+				observer,
+				oc.zetacoreClient,
+				zetaHeight,
+			)
+		}
+	}
+}
+
+// ScheduleCctxBTC schedules bitcoin outbound keysign on each ZetaChain block (the ticker)
+func (oc *Orchestrator) ScheduleCctxSolana(
```

Comment:
Why do we need different methods per chain for scheduling cctxs?

The orchestrator should be chain agnostic. We should create an issue and track it to refactor this.

Reply:
It's actually implemented in this PR https://github.com/zeta-chain/node/pull/2568/files#diff-1aa41ba28b878033f3bdb5e654e32205f1d99f1a21485ab6d297969cd426f280R357

Reply:
Let's track it with a issue: https://github.com/zeta-chain/node/issues/2613 
There are custom logic for each chain type (Bitcoin is different, EVM has some custom logic but it's still similar to Solana at the moment), we'll need to look more closely to come up with a generalized flow.